client/components/notice/notice-action.jsx

```diff
@@ -23,6 +24,12 @@ export default React.createClass( {
 		};
 	},
 
+	componentDidMount() {
+		if ( this.props.onMount ) {
+			this.props.onMount();
```

Comment:
Was thinking of a more generic metrics component wrapper if we do want to track when certain components are rendered